pkg/harness/environment/environment.go

```diff
@@ -38,7 +36,7 @@ func (in *environment) prepareFiles() error {
 	}
 
 	for _, fragment := range in.stackRun.Files {
-		destination := path.Join(in.filesDir, fragment.Path)
```

Comment:
IIUC it will require user to provide full path including harness working dir, i.e. `/plural/tf-test/...`. Isn't it more prone to errors? Any other root level dir than `/plural` will not work currently.

Reply:
You should be able to specify `.` as the mount path, and then it would be something like `./creds.json`.  I do think we should probably make some other folders writable though i just can't think of which at the moment (also think there probably will need to be a home folder for the user).

The problem I faced though was I didn't know the path at config-time at first, so it was a bit tricky to set things like GOOGLE_APPLICATION_CREDENTIALS properly, if you have full understanding of the path that's more straightforward.